src/NUnitFramework/tests/Constraints/RangeConstraintTests.cs

```diff
@@ -87,5 +87,77 @@ public void ShouldThrowExceptionIfFromIsLessThanTo()
         {
             Assert.Throws<ArgumentException>(() => new RangeConstraint( 42, 5 ));
         }
+
+        #if !NETCF && !SILVERLIGHT  && !PORTABLE
```

Comment:
Why is this define needed? RangeConstraint works on all platforms, so it should be tested on all platforms.